src/main/java/uk/gov/hmcts/reform/cwrdapi/util/JsonFeignResponseUtil.java

```
@@ -32,6 +34,15 @@ private JsonFeignResponseUtil() {
         }
     }
 
+    public static List<Object> mapObjectToList(Response response, Object clazz) {
+        try {
+            JavaType type = json.getTypeFactory().constructCollectionType(List.class, (Class<?>) clazz);
+            return json.readValue(response.body().asReader(Charset.defaultCharset()), type);
+        } catch (Exception e) {
+            return new ArrayList<>();
```

we are suppressing exception here.

---

will throw custom exception. thanks